augur/dates/__init__.py

```diff
@@ -187,8 +196,8 @@ def get_numerical_dates(metadata:pd.DataFrame, name_col = None, date_col='date',
         dates = metadata[date_col].apply(
             lambda date: get_numerical_date_from_value(
                 date,
-                fmt,
-                min_max_year
+                fmt=fmt,
+                min_max_year=min_max_year
```

Comment:
This diff chunk seems unrelated.

Reply:
Fixed somewhere in [force-push](https://github.com/nextstrain/augur/compare/bb19de83f043a4d37315ac1eb3c233ac8bbaea20..dc5b0493d31a8f3cb8f4dfaa0870939960ca9469)